lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -4,7 +4,7 @@
     color: $white;
     margin: auto;
     max-width: 1440px;
-    padding: $base-spacing 0 $section-spacing;
+    padding: $base-spacing 0 12px;
```

Comment:
I actually just realized we don't need this change, the $base-spacing [below](https://github.com/DoSomething/phoenix/pull/7067#discussion-diff-79857731R165) solved it